data/json/mapgen/ranch_camp.json

```diff
@@ -547,14 +549,14 @@
         "..%.....................................................................",
         "..%.....................................................................",
         "..%.....................................................................",
-        "..%.....................................................................",
-        "..%................................................|--|..WWWWwWWWwWWW...",
-        "..%..............................................|-|;;|-|W;;;;rrr;;;W...",
-        "..%.............................................||;;;;;;||H;;;;;;;;rW...",
-        "..%.............................................|;;;;;;;;A;;;;;;;;;rW...",
-        "..%.............................................||;;;;;;||H;;;;;;;;rW...",
-        "..%..............................................|-|;;|-|W;;;;;;;;;;w...",
-        "..%................................................|--|..W;;;;;;;5;;W...",
+        "..%.................................................<...................",
+        "..%................................................2222..WWWWwWWWwWWW...",
+        "..%..............................................222//222W;;;;rrr;;;W...",
+        "..%.............................................22//////22H;;;;;;;;rW...",
+        "..%.............................................2////////A;;;;;;;;;rW...",
+        "..%.............................................22//////22H;;;;;;;;rW...",
+        "..%..............................................222//222W;;;;;;;;;;w...",
+        "..%................................................2222..W;;;;;;;5;;W...",
```

Comment:
This silo was clearly drawn to look circular in non-square ascii. You could make it an actual circle if you want. I don't think there are many oval-shaped silos out there.

Reply:
I don't mind addressing this in a new PR when I add the hayloft.  I know there are other silos that refer to this one, not sure how dependent they are on it atm.